_fluo-docs-1-2/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Apache Fluo documentation
+nodoctitle: true
+---
+
+# Apache Fluo {{ page.version }} documentation
```

Comment:
The following concept is not fully formed, just sharing for consideration and discussion.

Maybe we could remove this page and do the following.

 * Add Release notes and Javadocs to sidebar.
 * When linking to 1.2/ docs, maybe it just redirects to 1.2/design.  So when you go to the 1.2 docs it goes to the design docs and that shows as open in the sidebar.
 * The Fluo Tour link is that the top of every Fluo page.. so maybe a Tour link is not needed.